src/s1reader/s1_reader.py

```diff
@@ -360,17 +384,20 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
     #    tree_lads = ET.parse(f_lads)
     #    product_annotation = s1_annotation.ProductAnnotation.from_et(tree_lads)
 
-    #load the Calibraton annotation
-    calibration_annotation_path = annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
-    with open_method(calibration_annotation_path, 'r') as f_cads:
-        tree_cads = ET.parse(f_cads)
-        calibration_annotation = s1_annotation.CalibrationAnnotation.from_et(tree_cads)
+    eap = is_eap_correction_necessary(ipf_version)
+    if eap.magnitude_correction or eap.magnitude_correction:
```

Comment:
```suggestion
    if eap.magnitude_correction or eap.phase_correction:
```
`magnitude_correction` checked twice. Should `phase_correction` also be `True`?

Reply:
Yes, this should be `phase_correction`. Thank you @LiangJYu!